README.md

```diff
@@ -2,9 +2,19 @@
 
 A tool for GUI automation using a variety of computer vision and desktop control backends.
 
+## Introduction
+
+In order to do GUI automation you usually need to solve two problems: first, you need to have a way to control and interact with the interface and platform you are automating and second, you need to be able to locate the objects you are interested in on the screen. Guibot helps you do both.
+
+To interact with GUIs, guibot provides the [`desktopcontrol`](https://github.com/intra2net/guibot/blob/master/guibot/desktopcontrol.py) module which contains a common interface for different desktop control backends, with methods to move the mouse, take screenshots, type characters and so on. The backend to use will depend on how your interface is accessible, but the ones currently supported are qemu, vnctotool, xdotool, and autopy.
```

Comment:
I am still wondering a lot about the `DesktopControl` class and if should be something like `ScreenController` or better. Especially if we want to change the summary.

Reply:
So I guess one rename here and this is solved.

Reply:
What rename exactly? I've capitalized the word "Guibot" in the newer version, this diff is outdated

Reply:
This thread is about `DesktopControl` and it is also what I am referring to.

Reply:
The rename of `DesktopControl` will not be part of this PR as this is restricted to extending `README.md`. Renaming it will take more work and will include renaming [configuration values](https://github.com/intra2net/guibot/blob/master/guibot/config.py#L306), [class attributes](https://github.com/intra2net/guibot/blob/master/guibot/region.py#L88), adding tests and so on. It won't be as simple as the `Path` one.

Reply:
As this PR is more maintenance related, I am planning to simply rebase it similarly to #17 this this won't matter. For the sake of PR clarity it should also be fine since renaming the module is with the purpose of readability and proper documentation. The only thing that will matter for me personally will be for the README commit to be last in the PR (on top) with all class names as current as possible.

Reply:
Regarding the renamed attributes and config values, perhaps we could leave out a dummy `DesktopControl` class that wraps around the new class returning the corresponding attributes with their old names. Would that be simpler? If not then we could go the separate PR path.

Reply:
> Would that be simpler? If not then we could go the separate PR path.

A separate PR, definitely. But for two reasons: I need to resume other tasks and it's been two weeks since the first PR of this batch, I don't want to keep dragging this any longer. And second, I consider the rename out of the scope of this PR. The PR meant to rename a class can fix the README, but a PR meant to extend the README shouldn't be renaming classes.

But of course, if you want you can take this over.

Reply:
Alright, let's keep it as a separate PR. I will prioritize it before this one to leave this one last. I hope tomorrow we will be in a good shape to end this :)

Reply:
Alright, that's great. Thanks!